services/prometheus/scrape_configs.go

```diff
@@ -174,68 +175,178 @@ func scrapeConfigForStandardExporter(interval time.Duration, node *models.Node,
 	return cfg, nil
 }
 
-func scrapeConfigForNodeExporter(interval time.Duration, node *models.Node, agent *models.Agent) (*config.ScrapeConfig, error) {
-	return scrapeConfigForStandardExporter(interval, node, nil, agent, []string{})
+func scraperConfigsForNodeExporter(s *models.MetricsResolutions, node *models.Node, agent *models.Agent) ([]*config.ScrapeConfig, error) {
+	hrc := []string{
+		"diskstats",
+		"filefd",
+		"filesystem",
+		"loadavg",
+		"meminfo",
+		"netdev",
+		"netstat",
+		"stat",
+		"time",
+		"vmstat",
+		"meminfo_numa",
+		"textfile.hr",
+		"textfile.directory.hr",
```

Comment:
Added `standard.process` (`prometheus.NewProcessCollector`) and `standard.go` (`prometheus.NewGoCollector`) to the Confluence page. When `collect[]` is present, and we filter collectors, we should enable them in exporter only when requested, and we should request them only for HR metrics in PMM. It is a mistake to send them for all resolutions.